nilearn/reporting/data/css/partials/navbar.jinja

```diff
@@ -0,0 +1,77 @@
+/* -----------------------  Navbar  ----------------------- */
```

Comment:
This seems like a css file, is the extension `jinja` a necessity?

Reply:
yeah for this one we could keep the css extension